src/pages/markets/stock/_digital-options.js

```diff
@@ -304,15 +122,15 @@ const DigitalOptions = () => {
                     />
                     <MarketsAccordion
                         renderTitle={() => (
-                            <Row jc="flex-start" ai="center">
+                            <Row is_accordion_row={true}>
                                 <Col>
-                                    <Title weight="bold" max_width="9.7rem">
-                                        {localize('Asia/ Oceania')}
-                                    </Title>
+                                    <Title>{localize('Asia/ Oceania')}</Title>
                                 </Col>
                                 <MarketsList
-                                    columns="repeat(3, 1fr)"
-                                    mobile_col_template="repeat(1, 1fr)"
+                                    col={3}
+                                    tablet_col={1}
```

Comment:
```suggestion
                                    tablet_col={2}
```

Reply:
Personally, I think it looks a lot better with 2 columns for tablet, even though the layout currently for production for tablet is 1 column as well.

Reply:
I agree. We can go for `minimum 2` for tablet if `mobile <= 2` otherwise will match the mobile 👍🏻